Update staff-recruiting/README.md

natalie-hub opened the pull request:
Removed the portions of the process that I moved to the new Recruiting Cadences page

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

natalie-hub committed:
Update staff-recruiting/README.md

Updated the Recruiting Process

natalie-hub committed:
Update staff-recruiting/README.md

Removed the portions of the process that I moved to the new Recruiting Cadences page

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
 ## Role Description 
 
-Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the in-take template. 
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule a [Recruiter In-Take Session](https://handbook.mattermost.com/contributors/join-us/staff-recruiting/Recruit ...
```
```suggestion
Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule a [Recruiter Intake Session](https://handbook.mattermost.com/contributors/join-us/staff-recruiting/recruiting-cadences.html) to review the remainder of the intake template. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
-Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule time to review the remainder of the in-take template. 
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule a [Recruiter In-Take Session](https://handbook.mattermost.com/contributors/join-us/staff-recruiting/Recruit ...
 
 Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
```
```suggestion
Your Recruiter is assigned based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
+Once you have confirmation that your new headcount has been approved, you’re ready to begin the recruitment process. Complete the [Job Description portion](https://docs.google.com/document/d/1rpTI2NKu4H_781vpx_eD9Fku7go4gGDs1P7bJAMQvAU/edit#) and share with your Recruiter, who will schedule a [Recruiter In-Take Session](https://handbook.mattermost.com/contributors/join-us/staff-recruiting/Recruit ...
 
 Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
 * Technical Individual Contributor Roles -- Teresa Novoa 
```
```suggestion
* Technical individual contributor roles - Teresa Novoa
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
 
 Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
 * Technical Individual Contributor Roles -- Teresa Novoa 
 * All others -- Natalie Jew 
```
```suggestion
* All others - Natalie Jew
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
-
 ### Role Leveling 
 
 You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). Note: HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job req, your recruiter can help you walk through this as part of the Recruiter In-Take. 
```
```suggestion
You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). 
**Note:** HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job requisition, your Recruiter can help you walk through this as part of the Recruiter Intake. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -129,14 +125,6 @@ In order to expedite the recruiting process for each candidate, please follow th
 * Feedback on the specific skill/focus area assigned with examples to help contextualize your rating  
 * Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown m&ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.
```
```suggestion
* Potential [blindspots](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) or [brown M&Ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) that may require either a future interviewer to ask about or the Hiring Manager to cover during a reference check.
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -129,14 +125,6 @@ In order to expedite the recruiting process for each candidate, please follow th
-* If you are an interviewer and cannot make the scheduled sync, please make sure to submit complete feedback in Lever ahead of the scheduled sync. 
-
 ### Interview Project
 Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the in-take. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 
```
```suggestion
Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the intake. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
 
 ## Role Description 
```
```suggestion
## Role description 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -11,16 +11,12 @@ Hiring Managers should confirm with your Team's MLT that your new headcount requ
-
-Your Recruiter will schedule an in-take session after you’ve completed your Job Description. During this call, your Recruiter will work with you to ensure alignment on the level, ideal candidate profile, the interview process and next steps. 
-
 ### Role Leveling 
```
```suggestion
### Role leveling 
```

justinegeffen commented on contributors/join-us/staff-recruiting/README.md:
```diff
@@ -129,14 +125,6 @@ In order to expedite the recruiting process for each candidate, please follow th
-* If there are multiple candidates for the same role, the sync may be scheduled after all candidates have gone through the last mid-interviewer. 
-* If you are an interviewer and cannot make the scheduled sync, please make sure to submit complete feedback in Lever ahead of the scheduled sync. 
-
 ### Interview Project
```
```suggestion
### Interview project
```

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update README.md

Added the link for MLT that Jason recommended.

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
Update contributors/join-us/staff-recruiting/README.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub commented:
Thank you @justinegeffen for making these changes!

natalie-hub committed:
Create Recruiting Cadences (#339)

* Create Recruiting Cadences

Created a Recruiting Cadence to help outline the Manager's responsibilities a bit more clearly.

* Update and rename Recruiting Cadences to recruiting-cadences.md

First, full editorial pass for new section.

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

justinegeffen committed:
Merge branch 'natalie-hub-patch-3' into natalie-hub-patch-6

justinegeffen committed:
Update README.md

justinegeffen commented:
@natalie-hub, I have a feeling I may have messed up your changes due to some conflicts I needed to resolve prior to merging. Please could you take a look through this and let me know whether it looks OK to you! Thanks!

natalie-hub commented:
This all looks good, sorry Justine - I think I'm behind on GitHub
responses.

On Tue, Jul 7, 2020 at 3:05 AM Justine Geffen <notifications@github.com>
wrote:

> @natalie-hub <https://github.com/natalie-hub>, I have a feeling I may
> have messed up your changes due to some conflicts I needed to resolve prior
> to merging. Please could you take a look through this and let me know
> whether it looks OK to you! Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mattermost/mattermost-handbook/pull/340#issuecomment-654746081>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANFSFBQ2UTIJ3AWTK4ZUNNLR2LXPXANCNFSM4OPMUK5Q>
> .
>


-- 
Natalie Jew
Director of HR, Mattermost, Inc.
natalie.jew@mattermost.com <alice@mattermost.com> | (415) 509-6750 |
mattermost.com

justinegeffen committed:
Merge branch '0.2.1' into natalie-hub-patch-6
